schema/bom-1.6.schema.json

```diff
@@ -1461,8 +1461,8 @@
       "additionalProperties": false,
       "properties": {
         "identity": {
-          "type": "object",
-          "description": "Evidence that substantiates the identity of a component.",
+          "type": ["array", "object"],
```

Comment:
An array with no items, but properties? 
This is not correct.

Reply:
it could be something like (untested pseudo code):

1. move current definition of `identity` a definition
2. change `identity` to 
   ```json
  { "description": "Evidence that substantiates the identity of a component.",
      "oneOf" : [
  { "type": "array", "items": { "$ref": "#/definitions/EvidenceIdentity" } },
  { "$ref": "#/definitions/EvidenceIdentity" }
  }]
}
```

Reply:
Good catch. Corrected.